educe/stac/context.py

```diff
@@ -92,22 +166,26 @@ def _the(cls, edu, surrounders, typ):
                 raise Exception(oops)
 
     @classmethod
-    def _for_edu(cls, enclosure, doc_turns, edu):
+    def _for_edu(cls, enclosure, doc_turns, doc_tstars, edu):
```

Comment:
Improvements on the docstring (especially parameters) would be welcome here too.